### ISSUE TITLE
The G logo should go to the configured git host home page

### ISSUE THREAD

**Issue by Juice805** [manosim/gitify#543]
Currently there are 2 buttons which point to this repo: The G logo, and the GitHub logo at the bottom of the sidebar. The logo should open the home page of the configured git host instead.

**Comment by pjconnors** [comment]
I think that it's appropriate for the G logo to go to the main [Gitify webpage](https://www.gitify.io/) (or this Github repo as it does now.)

But instead, link the GitHub icon at the bottom to the users' [Github Notifications page](https://github.com/notifications).  This icon, lives in the the area that's focused on the user along with the refresh and settings buttons.

The refresh button looks to the notification feed, but only displays unseen alerts.  A link to this page here gives direct access to the full notification history.

**Comment by hughlilly** [comment]
I agree that the Gitify logo ought to (continue) to go to the Gitify repo, and that the GitHub logo should go to //github.com/notifications.
